### DISCUSSION TITLE
Question 97: How to upload plant sightings to EMu [ZK - Action]

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#697] [Q&A] [answered]
Description: I have some plant observations (sightings) data from a recent RI that I’m getting ready to upload to EMu. I’m wondering, should I be uploading these records the same way as we’ve done sightings for different bio groups, using ActionMgr as the user group? Or should I upload these in a way that’s more similar to how Action does plant specimen records, using the RRC user group? [Here is a link to the spreadsheet](https://docs.google.com/spreadsheets/d/1ZP2iPIueBJd31wI8HLFaOKGFr-pwVspm/edit?usp=sharing&ouid=111065544285812031709&rtpof=true&sd=true) if it’s helpful.

**Comment by rondlg** [comment, marked as answer]
Response: The RRC user group workflow was developed because Botany’s specimens are being used by Action. In the case of sightings records Action is wholly responsible for the data. You should be able to use the workflow as with other collections, but there may be some adjustments that need to be made. 

Do specimens also exist for any of the botany sightings?
No, just sightings. No specimen or photos.
Zach will highlight and comment on the spreadsheet (linked above) for specific questions.

**Comment by XuDA18** [comment]
您好，来信已收到，我会尽快回复，谢谢！
